src/components/accordion/accordion.style.js

```diff
@@ -82,9 +82,15 @@ const StyledAccordionContentContainer = styled.div`
   box-sizing: border-box;
   overflow: hidden;
   transition: all 0.3s;
-  max-height: ${({ maxHeight }) => maxHeight}px;
-  height: ${({ maxHeight }) => maxHeight}px;
-  ${({ isExpanded }) => !isExpanded && 'visibility: hidden'};
+  ${({ maxHeight }) => css`
```

Comment:
Please can we change this. Currently a closed accordion will output two CSS rules for `max-height` and `height`.

Reply:
Good point will update